yogstation/code/modules/admin/verbs/one_click_antag.dm

```diff
@@ -0,0 +1,25 @@
+//Shadowling
+/datum/admins/proc/makeShadowling()
+	var/datum/game_mode/shadowling/temp = new
+	if(CONFIG_GET(flag/protect_roles_from_antagonist))
+		temp.restricted_jobs += temp.protected_jobs
+	if(CONFIG_GET(flag/protect_assistant_from_antagonist))
+		temp.restricted_jobs += "Assistant"
+	var/list/mob/living/carbon/human/candidates = list()
+	var/mob/living/carbon/human/H
+	for(var/mob/living/carbon/human/applicant in GLOB.player_list)
+		if(ROLE_SHADOWLING in applicant.client.prefs.be_special)
+			if(!applicant.stat)
+				if(applicant.mind)
+					if(!applicant.mind.special_role)
+						if(!jobban_isbanned(applicant, "shadowling") && !jobban_isbanned(applicant, "Syndicate"))
+							if(temp.age_check(applicant.client))
+								if(!(applicant.job in temp.restricted_jobs))
+									if(!(is_shadow_or_thrall(applicant)))
```

Comment:
Please stop the nesting.

Reply:
this thing doesnt even work, low priority that i'll fix later